Model compile apparent error in the getting started but fitting is successful?

The getting started vignette has a bug on main currently, see [here](https://epidist.epinowcast.org/articles/epidist.html#fit), or:

```r
fit <- epidist(
  data = data, chains = 2, cores = 2, refresh = ifelse(interactive()), 250, 0)
)
#> Running /opt/R/4.4.2/lib/R/bin/R CMD SHLIB foo.c
#> using C compiler: ‘gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0’
#> gcc -I"/opt/R/4.4.2/lib/R/include" -DNDEBUG   -I"/home/runner/work/_temp/Library/Rcpp/include/"  -I"/home/runner/work/_temp/Library/RcppEigen/include/"  -I"/home/runner/work/_temp/Library/RcppEigen/include/unsupported"  -I"/home/runner/work/_temp/Library/BH/include" -I"/home/runner/work/_temp/Library/StanHeaders/include/src/"  -I"/home/runner/work/_temp/Library/StanHeaders/include/"  -I"/home/runner/work/_temp/Library/RcppParallel/include/"  -I"/home/runner/work/_temp/Library/rstan/include" -DEIGEN_NO_DEBUG  -DBOOST_DISABLE_ASSERTS  -DBOOST_PENDING_INTEGER_LOG2_HPP  -DSTAN_THREADS  -DUSE_STANC3 -DSTRICT_R_HEADERS  -DBOOST_PHOENIX_NO_VARIADIC_EXPRESSION  -D_HAS_AUTO_PTR_ETC=0  -include '/home/runner/work/_temp/Library/StanHeaders/include/stan/math/prim/fun/Eigen.hpp'  -D_REENTRANT -DRCPP_PARALLEL_USE_TBB=1   -I/usr/local/include    -fpic  -g -O2  -c foo.c -o foo.o
#> In file included from /home/runner/work/_temp/Library/RcppEigen/include/Eigen/Core:19,
#>                  from /home/runner/work/_temp/Library/RcppEigen/include/Eigen/Dense:1,
#>                  from /home/runner/work/_temp/Library/StanHeaders/include/stan/math/prim/fun/Eigen.hpp:22,
#>                  from <command-line>:
#> /home/runner/work/_temp/Library/RcppEigen/include/Eigen/src/Core/util/Macros.h:679:10: fatal error: cmath: No such file or directory
#>   679 | #include <cmath>
#>       |          ^~~~~~~
#> compilation terminated.
#> make: *** [/opt/R/4.4.2/lib/R/etc/Makeconf:195: foo.o] Error 1
```

Looks to be to do with Stan versions ("fatal error: cmath: No such file or directory").

Edit: doesn't happen with `cmdstanr` backend so an `rstan` problem.

--- a/NEWS.md
+++ b/NEWS.md
@@ -4,7 +4,7 @@ This is the development version of `epidist`.
 
 ## Package
 
-- Remove caching of vignettes.
+- Remove caching of vignettes. See #533.
 
 # epidist 0.2.0
 

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,6 +1,11 @@
 # epidist 0.2.0.1000
 
 This is the development version of `epidist`.
+
+## Package
+
+- Remove caching of vignettes.
+
 # epidist 0.2.0
 
 This release adds a new marginal model based on `primarycensored` which provides a more efficient approach for fitting delay distributions compared to the existing latent model. We've also improved data handling by adding support for aggregated data across all models, added comprehensive examples using real world data, and enhanced documentation based on user feedback. The package has also undergone significant internal improvements including generalised Stan reparameterisation and improved data transformation methods.

--- a/vignettes/approx-inference.Rmd
+++ b/vignettes/approx-inference.Rmd
@@ -24,7 +24,6 @@ set.seed(2)
 
 knitr::opts_chunk$set(
   fig.path = file.path("figures", "epidist-"),
-  cache = TRUE,
   collapse = TRUE,
   comment = "#>",
   message = FALSE,

--- a/vignettes/ebola.Rmd
+++ b/vignettes/ebola.Rmd
@@ -21,13 +21,11 @@ bibliography: references.bib
 # exclude compile warnings from cmdstanr
 knitr::opts_chunk$set(
   fig.path = file.path("figures", "epidist-"),
-  cache = TRUE,
   collapse = TRUE,
   comment = "#>",
   message = FALSE,
   warning = FALSE,
-  error = FALSE,
-  cache.lazy = FALSE
+  error = FALSE
 )
 ```
 

--- a/vignettes/epidist.Rmd
+++ b/vignettes/epidist.Rmd
@@ -21,7 +21,6 @@ bibliography: references.bib
 # exclude compile warnings from cmdstanr
 knitr::opts_chunk$set(
   fig.path = file.path("figures", "epidist-"),
-  cache = TRUE,
   collapse = TRUE,
   comment = "#>",
   message = FALSE,

